packages/components/filter-bar/pipes/pipe-multi-select.ts

```diff
@@ -122,9 +122,9 @@ export class KbqPipeMultiSelectComponent extends KbqBasePipe<KbqSelectValue[]> i
 
     /** @docs-private */
     ngOnInit(): void {
-        this.filteredOptions = merge(
-            of(this.values),
-            this.searchControl.valueChanges.pipe(map((value) => this.getFilteredOptions(value)))
+        this.filteredOptions = merge(this.filterBar!.internalTemplatesChanges, this.searchControl.valueChanges).pipe(
```

Comment:
зачем восклицательный знак? проперти же не опциональное, а если опциональное, то это потенциальное место для рантайм ошибки
```suggestion
        this.filteredOptions = merge(this.filterBar.internalTemplatesChanges, this.searchControl.valueChanges).pipe(
```

Reply:
Сейчас `filterBar` опциональный. Изначально задумка была использовать пайпы отдельно, но в процессе получилось два `сильно` связанных компонента. Ошибка в рантайме может быть, но мы пока не заявляли о таком функционале, поэтому мало вероятно. Порефакторить нужно будет, да...